manage.py

```diff
@@ -6,7 +6,7 @@
     if 'test' in sys.argv:
         os.environ['DJANGO_SETTINGS_MODULE'] = 'icommons_ext_tools.settings.test_settings'
     else:
-        os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'icommons_ext_tools.settings.local')
+        os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'icommons_ext_tools.settings.aws')
```

Comment:
I think it would be more beneficial if it defaulted to AWS as the wsgi file does so that we don't have to set the `DJANGO_SETTINGS_MODULE` in every management command we want to run during deployment.  I suppose once _every_ project is converted to using AWS